nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/CopyAzureBlobStorage_v12.java

```diff
@@ -0,0 +1,388 @@
+            .name("source-container-name")
+            .displayName("Source Container Name")
+            .description("Name of the Azure storage container to copy from.")
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .required(true)
+            .build();
+
+    public static final PropertyDescriptor SOURCE_BLOB_NAME = new PropertyDescriptor.Builder()
+            .name("source-blob-name")
+            .displayName("Source Blob Name")
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .required(true)
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .description("The full name of the source blob")
+            .build();
+
+    public static final PropertyDescriptor DESTINATION_STORAGE_CREDENTIALS_SERVICE = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(STORAGE_CREDENTIALS_SERVICE)
+            .displayName("Destination Storage Credentials")
+            .build();
+
+    public static final PropertyDescriptor DESTINATION_CONTAINER = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(AzureStorageUtils.CONTAINER)
+            .displayName("Destination Container Name")
+            .description("Name of the Azure storage container to copy into; defaults to source container name.")
+            .required(false)
+            .build();
+
+    public static final PropertyDescriptor DESTINATION_BLOB_NAME = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(BLOB_NAME)
+            .displayName("Destination Blob Name")
+            .description("The full name of the destination blob; defaults to source blob name.")
+            .required(false)
+            .build();
+
+    private static final List<PropertyDescriptor> PROPERTIES = Collections.unmodifiableList(Arrays.asList(
+            SOURCE_STORAGE_CREDENTIALS_SERVICE,
+            SOURCE_CONTAINER,
+            SOURCE_BLOB_NAME,
+            DESTINATION_STORAGE_CREDENTIALS_SERVICE,
+            DESTINATION_BLOB_NAME,
+            DESTINATION_CONTAINER,
+            AzureStorageUtils.CONFLICT_RESOLUTION,
+            AzureStorageUtils.CREATE_CONTAINER,
+            AzureStorageUtils.PROXY_CONFIGURATION_SERVICE
+    ));
+
+    @Override
+    protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
+        return PROPERTIES;
+    }
+
+    @OnScheduled
+    public void onScheduled(ProcessContext context) {
+        super.onScheduled(context);
+    }
+
+    @OnStopped
+    public void onStopped() {
+        super.onStopped();
+    }
+
+    private static AzureStorageCredentialsService_v12 getCopyFromCredentialsService(ProcessContext context) {
+        return context.getProperty(SOURCE_STORAGE_CREDENTIALS_SERVICE).asControllerService(AzureStorageCredentialsService_v12.class);
+    }
+
```

Comment:
Add `@Override` to this method.
```suggestion
    @Override
```